apps/fetch.py

```diff
@@ -71,20 +74,27 @@ def main(args: argparse.Namespace):
         with open(args.targets_file) as f:
             targets = json.load(f)
 
-        shortlist = [x.strip() for x in args.apps_shortlist.split(',') if x] \
-            if args.apps_shortlist else None
-
-        fetched_target_apps = fetch_target_apps(targets, shortlist, token, args.fetch_dir)
-        for target, target_json in targets.items():
-            out_file = os.path.join(args.dst_dir, f"{target}.apps.tar")
-            logging.info(f"Tarring fetched apps of {target} to {out_file}...")
-            tar_fetched_apps(os.path.join(args.fetch_dir, target), out_file)
-            target_json["custom"]["fetched-apps"] = {
-                "uri": os.path.join(os.environ["H_RUN_URL"], f"{target}.apps.tar"),
-            }
-            if fetched_target_apps[target] and len(fetched_target_apps[target]) > 0:
-                target_json["custom"]["fetched-apps"]["shortlist"] = \
-                                                            ",".join(fetched_target_apps[target])
+        if args.disable:
+            logging.info("Disabling apps for offline update...")
+            for target, target_json in targets.items():
+                target_json["custom"]["fetched-apps"] = {
+                    "uri": None
+                }
```

Comment:
would it make more sense to do:
` target_json["custom"]["fetched-apps"]  = None`
or
` target_json["custom"]["fetched-apps"]  = {}`

Reply:
Maybe, it will require a change in aklite though and I've already update the meta-lmp... This is kind of internal interface between ci-scripts and aklite, not user facing stuff.